### PR TITLE
Update scrna page

### PR DESCRIPTION
**Pull request by mtekman**
Added missing tutorials, need to find some histories for them too.

Also plan to update the tools using a table format using attached table converted into markdown

[single_cell_tools_feature_comparison.xlsx](https://github.com/usegalaxy-eu/website/files/4031223/single_cell_tools_feature_comparison.xlsx)

### CODE REVIEW

File: index-singlecell.md [hunk 1]
```diff
@@ -52,10 +54,43 @@ In this section we list all tools that have been integrated in the the single ce
...
+|  DESeq2 | CSV htseq-count summarizedexperiment | Y         | DESeq2                                    |                                    | Y                        |                                     |                    |                         |                                                                                 |
+|  scPipe |                                      | Y         | Y                                         | Y                                  | Y                        | Y                                   |                    |                         | CelSeq2, etc. Uses Scater and SCRAN for downstream. Also perform pre-processing |
+
+
+
 Tool | Description | Reference
 --- | --- | ---
 {% include tool.html id="FastQC" %} | A quality control tool for high throughput sequence data | -
-{% include tool.html id="MultiQC" %} | MultiQC aggregate results from bioinformatics analyses into a single report | [Ewels et al. 2016](https://doi.org/10.1093/bioinformatics/btw354){:target="_blank"}
+ | MQC | [Ewels et al. 2016](https://doi.org/10.1093/bioinformatics/btw354){:target="_blank"}
```
Comment (bgruening) [inline index-singlecell.md]: why this change?

**Comment by bgruening** [comment]
@mtekman is this still WIP?

File: index-singlecell.md (hunk 1 above)
Comment (mtekman) [reply index-singlecell.md]: It's used in Line62, so I was probably in the process of moving it.

**Comment by mtekman** [comment]
@bgruening very much so still a WIP -- it is just hard to write wide tables in markdown.

**Comment by bgruening** [comment]
It its not working ... what about an image? Or multiple small tables?

**Comment by mtekman** [comment]
The two table structure (preprocessing tools, downstream tools) makes sense,  I just need to update the descriptions and references. A few hours of work, I think

**Comment by mtekman** [comment]
Complete, but likely full of `{% include tool.html id=<blah> %}` errors since I am not sure whether these ids related to folder names in tools-iuc, or reference the shed.yml name.

**Comment by bgruening** [comment]
xref: https://github.com/usegalaxy-eu/website/blob/master/_data/tools.yml

**Comment by mtekman** [comment]
Thanks, will update
